spec/reek/smells/smell_warning_spec.rb

```diff
@@ -110,7 +120,15 @@
 
     it 'does not match on different params' do
       expect(uncommunicative.matches?(:UncommunicativeVariableName,
-                                      test: 'something else')).to be_falsy
+                                      parameters: {
+                                        test: 'something else'
+                                      })).to be_falsy
+    end
+
+    it 'does not match on different vars' do
+      expect(uncommunicative.matches?(:UncommunicativeVariableName,
+                                      parameters: { test: 'something' },
+                                      message: 'nothing')).to be_falsy
     end
```

Comment:
I had a thought already about how we are not testing attribute match failure and had already written a test for that which I thought to include in a subsequent pull request.  I include that now, although that is still in the SmellWarning#matches? context.  Various of the code paths of SmellWarning#matches_attributes? are tested via the SmellWarning#matches? context.  Should they be broken out into a separate context? tested in both (no I guess) or should SmellWarning#matches_attributes? become a private method?